wagtail/users/views/users.py

```diff
@@ -180,57 +182,72 @@ def create(request):
     )
 
 
-@permission_required(change_user_perm)
-def edit(request, user_id):
-    user = get_object_or_404(User, pk=user_id)
-    can_delete = user_can_delete_user(request.user, user)
-    editing_self = request.user == user
+class Edit(EditView):
+
+    model = User
+    permission_policy = ModelPermissionPolicy(User)
+    form_class = get_user_edit_form()
+    template_name = "wagtailusers/users/edit.html"
+    index_url_name = "wagtailusers_users:index"
+    success_url = "wagtailusers_users:index"
+    edit_url_name = "wagtailusers_users:edit"
+    delete_url_name = "wagtailusers_users:delete"
+    success_message = _("User '{0}' updated.")
+    context_object_name = "user"
+
+    def get_page_title(self):
+        return "Editing {}".format(self.object.get_username)
```

Comment:
This does not work as expected, firstly the `get_username` must be called, secondly we should try to use the other translation syntax as it works better for translators.

Finally, when we do this, we need to also stop the subtitle from rendering as it duplicates the instance name.

```suggestion
        return _("Editing %s") % self.object.get_username()

    def get_page_subtitle(self):
        return ""
```

<img width="522" alt="Screen Shot 2022-10-04 at 3 28 16 pm" src="https://user-images.githubusercontent.com/1396140/193743506-9764cb69-7def-47c2-a13b-5f6469203cce.png">